src/ir/var.rs

```diff
@@ -169,3 +165,22 @@ fn parse_int_literal_tokens(cursor: &clang::Cursor,
         lit.parse().ok()
     }
 }
+
+fn get_integer_literal_from_cursor(cursor: &clang::Cursor,
+                                   unit: &clang::TranslationUnit) -> Option<i64> {
+    use clangll::*;
+    let mut value = None;
+    cursor.visit(|c, _| {
+        match c.kind() {
+            CXCursor_IntegerLiteral => {
+                value = parse_int_literal_tokens(&c, unit);
+            }
+            CXCursor_FirstExpr => {
```

Comment:
Can you use `UnexposedExpr` instead? First/Last is just the way clang gives you to iterate over them.